### ISSUE TITLE
Wrong command for saving screenshot

### ISSUE THREAD

**Issue by balukov** [webdriverio/webdriverio#3360]
**Pre-check**
- [x] I'm aware that I can [edit the docs](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) and submit a pull request

**Describe the improvement**

I'd like to report
- [x] Unclear documentation
- [ ] A typo
- [ ] Missing documentation
- [ ] Other

**Description of the improvement / report**

1. open doc: https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-allure-reporter/README.md
2. section "Add Screenshots"

It contains command `takeScreenshot`, but should `saveScreenshot`

**Comment by christian-bromann** [comment]
> Or if we want save in special folder too we can use saveScreenshot:
> var name = 'ERROR-chrome-' + Date.now()
> browser.saveScreenshot('./errorShots/' + name + '.png')

I like this solution, mind making a PR?

**Comment by xlenz** [comment]
@balukov @christian-bromann  How are you guys attaching screenshots to allure report? 

such config doesn't work for me
```
  reporters: [['allure', {
    outputDir: 'allure-results',
    disableWebdriverStepsReporting: true,
    disableWebdriverScreenshotsReporting: false,
  }]],
```

having addAttachment in afterTest hook throws an error saying that addAttachment is not a function. It would be helpful to update doc with this information as well

**Comment by xlenz** [comment]
ah, sorry, by bad
it should be 
```
  reporters: [['allure', {
    outputDir: 'allure-results',
    disableWebdriverStepsReporting: true,
    disableWebdriverScreenshotsReporting: false,
  }]],
```
in combination with
```
  afterTest: async function (test) {
    if (test.error !== undefined) {
      browser.takeScreenshot()
    }
  },
```

**Comment by christian-bromann** [comment]
@xlenz @balukov it seems that [`takeScreenshot`](https://webdriver.io/docs/api/webdriver.html#takescreenshot) is a protocol command that is called without doing anything with the result. Does our Allure reporter automatically pick this up?

**Comment by xlenz** [comment]
@christian-bromann having this in wdio.conf.js:
```
  afterTest: async function (test) {
    if (test.error !== undefined) {
      browser.takeScreenshot()
    }
  },
```
allure reporter is not making a screenshots.

And that is how it is designed
```
    isScreenshotCommand(command) {
        const isScrenshotEndpoint = /\/session\/[^/]*\/screenshot/
        return isScrenshotEndpoint.test(command.endpoint)
    }

    if (this.isScreenshotCommand(command) && command.result.value) {
            if (!disableWebdriverScreenshotsReporting) {
                this.allure.addAttachment('Screenshot', Buffer.from(command.result.value, 'base64'))
            }
        }
```

**Comment by christian-bromann** [comment]
> allure reporter is not making a screenshots.

I guess this is the bug then.

**Comment by BorisOsipov** [comment]
@christian-bromann it is not a bug :) 

@xlenz get screenshot in report
```
in such case there are screenshot and an attachment in allure result xml file

<attachments>
  <attachment title='Screenshot' source='some-uuid-attachment.png' type='image/png' size='93468'/>
</attachments>
```

by 

> allure reporter is not making a screenshots

@xlenz means that reporter isn't making screenshot itself. It gets base64 value from command result.

**Comment by BorisOsipov** [comment]
>however we need to capture screenshot manually in afterTest hook. I don't know if it is a bug or not

No, it isn't bug. Reporter can't make screenshots. You are responsible for doing it somehow when you need.

**Comment by balukov** [comment]
@BorisOsipov 
thank you, because I spent a lot of time to find this bug.
And because `disableWebdriverScreenshotsReporting: false` it's a default value I have just this settings in `wdio.conf.js` for screenshots:
```
  reporters: [
    'spec',
    [
      'allure',
      {
        disableWebdriverStepsReporting: true
      }
    ]
  ],
```
and
```
  afterTest: function(test) {
    if (test.error !== undefined) {
      browser.takeScreenshot();
    }
  }
```
